Dockerfile

```diff
@@ -9,8 +9,37 @@ RUN apk add --no-cache npm \
   && npm i /src \
   && npm run build
 
-FROM nginx:latest
+################################################
 
-COPY --from=build  /src/build /usr/share/nginx/html
+FROM alpine:3 as lighttpd-builder
+ARG LIGHTTPD_VERSION="1.4.77"
+# inspired by source build description:https://git.lighttpd.net/lighttpd/lighttpd1.4/src/branch/master/INSTALL
 
-ENTRYPOINT ["/bin/bash", "-c", "/usr/share/nginx/html/generate_config.sh && nginx -g \"daemon off;\""]
+ADD https://download.lighttpd.net/lighttpd/releases-1.4.x/lighttpd-$LIGHTTPD_VERSION.tar.gz \
+  /tmp/lighttpd.tar.gz
+
+RUN apk add \
```

Comment:
considering the build, it might be nice to mv into separate shellscript / makefile + we might consider keeping a separate image just w/ lighttpd (as a base-image; rationale: there is no need to keep re-building lighttpd each time we release a new version of delivery-dashboard; esp. assuming there will be very rare cases where we need a patch of lighttpd